src/Orleans.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
You can assign the people who runs the sqlutils test regularly to this PR. They are the people who care about the quality of sqlutils most. And they will be able to help you.